models/keep3r_network/ethereum/keep3r_network_ethereum_schema.yml

```diff
@@ -0,0 +1,107 @@
+version: 2
+
+models:
+  - name: keep3r_network_ethereum_view_job_migrations
+    meta:
+      blockchain: ethereum
+      project: keep3r_network
+      contributors: [agaperste, wei3erHase]
+    config:
+      tags: ["ethereum", "keep3r_network", "view"]
+    description: >
+      a view logging jobs migrations on Keep3r Network on Ethereum
```

Comment:
I feel like I still don't understand what this table is from this description. I'm going to come back to this but I'm looking on their site to see if I can suggest another descriptions. https://keep3r.network/